src/server.h

```diff
@@ -2911,6 +2950,10 @@ void runAndPropogateToReplicas(FN_PTR *pfn, TARGS... args) {
 }
 
 
+void killIOThreads(void);
```

Comment:
We don't have IO threads.  What about the other killThreads functions, are those io-thread specific?

Reply:
No, they aren't.